repo_e/square_of_tuple.py

```diff
@@ -0,0 +1,18 @@
+def square_tuple(tuple):
+    square = ()
+    for number in tuple:
+        square += (number**2,)
```

Comment:
that's not really a common approach, tuples are immutable (same as strings) and invoking `+=` on a tuple isn't a good practice

Reply:
I changed tuple into list